bazel/repositories.bzl

```diff
@@ -1,7 +1,7 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-ENVOY_COMMIT = "e1cd4cca46179562cf80de7f81774f4dc6354085"
-ENVOY_SHA = "16b63facab69d31accf06e6991627e7c6681749cae919c2d3c65217882127172"
+ENVOY_COMMIT = "d21d92ca21cd827e8ab484244a9eefc31bfd8af4"
```

Comment:
Consider adding the date of the comment as a comment.

Reply:
Added